sdk/examples/wallet/update_alias_output.rs

```diff
@@ -39,11 +37,11 @@ async fn main() -> Result<()> {
 
     // Set the stronghold password
     wallet
-        .set_stronghold_password(var("STRONGHOLD_PASSWORD").unwrap())
+        .set_stronghold_password(std::env::var("STRONGHOLD_PASSWORD").unwrap())
         .await?;
 
     // Get the alias output by its alias id
-    let alias_id = AliasId::from_str(ALIAS_ID)?;
+    let alias_id = ALIAS_ID.parse()?;
```

Comment:
IMO this is less readable, I'd rather we see the type

Reply:
okay, but that implies a `std` import? Fine with that?

Reply:
@thibault-martinez

Reply:
I don't like it but let's keep it like that so we can finally merge